src/spatial.rs

```diff
@@ -66,7 +66,7 @@ pub struct SpatialEmitter {
 impl Default for SpatialEmitter {
     fn default() -> Self {
         Self {
-            attenuation: None,
+            attenuation: Some(Easing::Linear),
```

Comment:
I think it would make more sense to use a quadratic ease-out, as attenuation follows the inverse square law; that being said it looks like Godot and Unreal default to linear attenuation as well, which feels weird to me, but lends credence to this default being standard.

```suggestion
            attenuation: Some(Easing::OutPowi(2)),
```

Reply:
I think your justification is good enough to go with your suggestion. We could probably go back and forth over which is best, but really we just want a non-`None` default and if users really need their own function they are going to change it anyway. There will be plenty of chances in the future to change the default to linear if there's a really good reason to.